tests/bluetooth/shell/testcase.yaml

```diff
@@ -289,14 +289,22 @@ tests:
     build_only: true
     platform_allow: native_posix
     extra_configs:
-      - CONFIG_BT_TBS_CLIENT=n
+      - CONFIG_BT_TBS_CLIENT_TBS=n
+      - CONFIG_BT_TBS_CLIENT_GTBS=n
```

Comment:
This would not be a valid config as per the TBS spec

Reply:
See my previous comment